_deployment/checklist/003_deploy_to_kubernetes.md

```diff
@@ -97,7 +98,7 @@ Type of information being modified
 - DB information
 - Redis
 - Rabbit
-- S3 information in tds-exam-service.yml
+- S3 information in tds-content-service.yml
```

Comment:
I'd like the common SQL stuff to go against a 3.x system.  We can follow up in a separate PR to point everything to the new 4.x system

Reply:
Looks like you just need to revert these changes.